### PR TITLE
added new api endpoint, create local user

### PR DESCRIPTION
**Pull request by iljaSL**

### CODE REVIEW

File: api/userstore/client.go [hunk 1]
```diff
@@ -26,6 +26,17 @@ func New(api restapi.Connector) *UserStore {
 	return &UserStore{api: api}
 }
 
+// CreateLocalUser create a new local PrivX user
+func (store *UserStore) CreateLocalUser(newUser interface{}) error {
```
Comment (kimmokssh) [inline api/userstore/client.go:30]: Should take defined LocalUser struct as parameter instead of interface.

File: api/userstore/client.go [hunk 2]
```diff
@@ -26,6 +26,17 @@ func New(api restapi.Connector) *UserStore {
...
+	req := newUser
+
+	_, err := store.api.
+		URL("/local-user-store/api/v1/users").
+		Post(&req)
+
+	return err
```
Comment (kimmokssh) [inline api/userstore/client.go:37]: The ID of created user should be returned if successful.

**Comment by iljaSL** [comment]
just a reminder post. :)

**Comment by iljaSL** [comment]
Function now takes the LocalUser as paramter and returns an UID if the creation was successful.
I also added Password to the LocalUser struct.

**Pull request merged**